template/src/store/selectors/loading.js

```diff
@@ -0,0 +1,14 @@
+export const getLoadingSelector = (state, actionTypes) => {
+  if (Array.isArray(actionTypes)) {
+    // some of element of list actionTypes is dispatched in redux will return state loading
```

Comment:
please remove this comment if not neccessary

Reply:
LGTM, i will remove this

Reply:
i did it, please review this again @huydosgtech